nncf/experimental/torch/fx/quantization/quantize_model.py

```diff
@@ -108,6 +116,8 @@ def quantize_impl(
     quantized_model.meta.update(original_graph_meta)
     quantized_model = _disallow_eval_train(quantized_model)
 
+    quantized_model = GraphModule(quantized_model, quantized_model.graph)
+
```

Comment:
Could you please add a comment here describing why we need to create a new `GraphModule`?

Reply:
Alright

Reply:
Done